applications/asset_tracker_v2/src/cloud/cloud_codec/json_protocol_names.h

```diff
@@ -166,6 +176,16 @@
 #define DATA_AGPS_REQUEST_TAC           "area"
 #define DATA_AGPS_REQUEST_TYPES         "types"
 
+#define DATA_AGPS_REQUEST_TYPE_UTC_PARAMETERS		 1
```

Comment:
Duplicate defines.

Reply:
A lot of these defines needs to be defined for both Azure, AWS and nRF Cloud builds so there might be duplicates. I don't see an issue with having only one set of defines for values that are identical across cloud backends. We could even have separate `_protocol_names.h` files for each cloud backend to make the abstraction clearer.

Reply:
Yes, I just followed the convention that had been used earlier. There's a lot of stuff which is defined twice in the same way depending on CONFIG_NRF_CLOUD_MQTT.

Should I do something in this PR?

Reply:
Its not really in the scope of the PR so I wouldn't worry about it.